networking/ptp/about-ptp.adoc

```diff
@@ -8,6 +8,12 @@ toc::[]
 
 Precision Time Protocol (PTP) is used to synchronize clocks in a network. When used in conjunction with hardware support, PTP is capable of sub-microsecond accuracy, and is more accurate than Network Time Protocol (NTP).
 
+[IMPORTANT]
+====
+If your `openshift-sdn` cluster with Precision Time Protocol (PTP) uses the User Datagram Protocol (UDP) for hardware time stamping, the hardware time stamping cannot be applied to primary interface devices, such as an Open vSwitch (OVS) bridge, after you migrate to the OVN-Kubernetes plugin. This means that 
```

Comment:
```suggestion
If your `openshift-sdn` cluster with PTP uses the User Datagram Protocol (UDP) for hardware time stamping and you migrate to the OVN-Kubernetes plugin, the hardware time stamping cannot be applied to primary interface devices, such as an Open vSwitch (OVS) bridge. As a result, UDP version 4 configurations cannot work with a `br-ex` interface.
```
- You should be able to use PTP here since you defined it earlier on this page.
- I tried to make the first sentence a little easier to read. I'm not sure if my revision is any better. I can see why all those details are needed, but it's a lot to take in.
- I removed the line wrap between lines 13 and 14.

Reply:
:heavy_check_mark: